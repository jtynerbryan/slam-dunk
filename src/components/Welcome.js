import React from 'react';
import Navbar from './Navbar'

class Welcome extends React.Component {

  render () {
    return (
      <div className="welcome">
        <Navbar />
        <div id="welcome-header">
          <h1 id="welcome-h1">Slam Dunk</h1>
          <h2 id="welcome-h2">The latest highlights from across the NBA</h2>
          <br></br>
        </div>
      </div>
    )
  }
}


export default Welcome
