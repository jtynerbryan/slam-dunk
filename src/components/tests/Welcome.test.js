import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import Welcome from '../Welcome';

describe('Welcome Component', () => {
  it('renders without crashing', () => {
    shallow(<Welcome />);
  });

  it('renders correctly', () => {
    const tree = renderer.create(<Welcome />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
