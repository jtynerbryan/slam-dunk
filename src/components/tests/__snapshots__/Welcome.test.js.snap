// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome Component renders correctly 1`] = `
<div
  className="welcome"
>
  <div
    className="ui inverted menu"
  >
    <a
      className="item"
      onClick={[Function]}
    >
      Slam Dunk
    </a>
    <a
      className="item"
      href="http://github.com/jtynerbryan"
      onClick={[Function]}
      target="_blank"
    >
      GitHub
    </a>
  </div>
  <div
    id="welcome-header"
  >
    <h1
      id="welcome-h1"
    >
      Slam Dunk
    </h1>
    <h2
      id="welcome-h2"
    >
      The latest highlights from across the NBA
    </h2>
    <br />
  </div>
</div>
`;
